### PR TITLE
#163179993 Setup Database

### PR DESCRIPTION
**Pull request by tunedev**
#### What does this PR do?
Setup Database using postgres
#### Description of Task to be completed?
Replace data structure with database.
#### Any background context you want to provide?
A data structure was used for the testing of endpoints before.
#### What are the relevant pivotal tracker stories?
[#163179993](https://www.pivotaltracker.com/story/show/163179993)

### CODE REVIEW

File: dist/app.js [hunk 1]
```diff
@@ -19,7 +19,9 @@ var _question = _interopRequireDefault(require("./routes/question"));
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
+// import dotenv from 'dotenv';
 // Import Routes
```
Comment (chukwuemekachm) [inline dist/app.js]: Please remove your dist files.

Comment (tunedev) [reply dist/app.js]: okay

Comment (tunedev) [reply dist/app.js]: i have removed it.

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Please update your commit message to follow the convention.

**Review by chukwuemekachm — CHANGES_REQUESTED** [review]
Please resolve the failing build.

**Pull request merged**